src/wrappers/any.jl

```diff
@@ -190,12 +196,27 @@ function start_worker(args=ARGS)
 
     parsed_args = parse_args(args, s)
 
-    open(joinpath(parsed_args["logs_dir"], "julia_worker.log"), "w+") do io
-        global_logger(SimpleLogger(io))
-        @info "Testing"
-        initialize_coreworker_worker(
-            parsed_args["node_manager_port"],
-            CxxWrap.@safe_cfunction(task_executor, Int32, ()),
-        )
-    end
+    # Note (omus): Logging is currently limited to a single worker as all workers attempt to
+    # write to the same file.
+    global_logger(FileLogger(joinpath(parsed_args["logs_dir"], "julia_worker.log");
+                             append=true, always_flush=true))
```

Comment:
I needed to add `FileLogger` (from LoggingExtras.jl`) here otherwise the file IO doens't flush on each write and you get no logs at all